source/fundamentals/indexes.txt

```diff
@@ -59,6 +59,17 @@ most common index types and provide sample code for creating each index type.
    The MongoDB Java Driver provides the :java-core-api:`Indexes </com/mongodb/client/model/Indexes.html>` class that
    includes static factory methods to create index specification documents for different MongoDB Index key types.
 
+The following examples use the
+:java-sync-api:`createIndex </com/mongodb/client/MongoCollection.html#createIndex(org.bson.conversions.Bson,com.mongodb.client.model.IndexOptions)>`
```

Comment:
Noticed something here that we struggle with in our Usage Examples for the Realm SDKs -- normally I like the idea of linking once to a concept at first mention. But for pages where individual sections stand alone as examples, I think it's sometimes useful to link one *per section*, especially since otherwise users might have to hunt through the javadoc to find the ``createIndex()`` method.

Reply:
Normally I'd want to link to the method more, but I think there are two mitigating things here:
1. The builders page, which included updating this page with links to it once it is complete, will serve as the primary page for "this is how you do indexes in Java".
2. In all of the IDEs I've tested, this method almost immediately completes and shows signature help.

I'm hesitant to add more items to the page that make it busier with these two things in mind. Thoughts?

Reply:
Seems fair to assume that the builder links and general norm of using an IDE with Java can fill this gap in understanding. LGTM